source/content/migrate-manual.md

```diff
@@ -12,7 +12,7 @@ integration: [--]
 
 Manually migrate your site to Pantheon when any of the following apply:
```

Comment:
@stovak , still contains multiple references to D9.  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/migrate-manual